tests/core/test_stuff.py

```diff
@@ -466,3 +468,21 @@ def is_in_calsteps(x: mass2.core.RecipeStep, steps: mass2.core.Recipe) -> bool:
 
     with pytest.raises(ValueError):
         steps.trim_dead_ends("this field doesn't exist")
+
+
+def test_save_analysis(tmpdir):
```

Comment:
Dude, this test feels nice and clean in a way that wouldn't have been possible with mass. Not quite sure why.

Reply:
Agree. I really like this one.